### PR TITLE
Add twice-daily schedule to update

### PR DESCRIPTION
**Pull request by eemeli**
This follows the [current practice](https://searchfox.org/mozilla-central/rev/5e0c42441a93d7a2307d3baa49d3a15f553d6757/.cron.yml#351-361) of checking for updates every day at 11:00 and 23:00 UTC.

### CODE REVIEW

**Comment by flodolo** [comment]
> This follows the [current practice](https://searchfox.org/mozilla-central/rev/5e0c42441a93d7a2307d3baa49d3a15f553d6757/.cron.yml#351-361) of checking for updates every day at 11:00 and 23:00 UTC.

We might need to tweak these, at least initially. These values were set because, at that point, the merge from `autoland` to `mozilla-central` should have been completed. 

How much longer does it take for the `gecko-dev` sync to happen?

**Comment by eemeli** [comment]
> How much longer does it take for the `gecko-dev` sync to happen?

This sounds like a question for @globau.

**Comment by globau** [comment]
> > How much longer does it take for the `gecko-dev` sync to happen?

https://mozilla-hub.atlassian.net/wiki/spaces/ROPS/pages/212468266/VCS+Sync

Every 4 minutes.

**Comment by flodolo** [comment]
🤯 Thanks, then I think we can keep the current schedule.

File: .github/workflows/update.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@
 name: Update
 
 on:
+  schedule:
+    - cron: "0 11,23 * * *"
```
Comment (flodolo) [inline .github/workflows/update.yml:9]:
```suggestion
    - cron: "0 11,23 * * 1-5"
```

Let's be nice and don't run on week-ends (although one can still trigger manually if they need it)

**Review by flodolo — APPROVED** [review]

**Pull request merged**